File: src/auth/42/user.session.dto.ts
export class UserSessionDto {
  /**
   * 42 계정 고유 ID
   */
  user_id: number;

  /**
   * 42 로그인 ID
   */
  intra_id: string;

  /**
   * 42 이메일
   */
  email: string;

  /**
   * 42 계정 전화번호
   */
  phone?: string;
}

File: backend/src/main/java/org/ftclub/cabinet/dto/UserSessionDto.java
package org.ftclub.cabinet.dto;

import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.ToString;

/**
 * 유저의 정보와 로그인 세션 정보를 갖는 DTO입니다.
 */
@AllArgsConstructor
@Getter
@ToString
public class UserSessionDto {

    private final Long userId;
    private final String name;
    private final String email;
    private final Integer tokenIssuedAt;
    private final Integer tokenExpiresAt;
    private final LocalDateTime blackHoledAt;
    private final Boolean isStaff;
}
